### PR TITLE
Duplicate fake blobid updatedate write fix

### PR DESCRIPTION
**Pull request by WhiteAu**
two bugs: 1st commit fixes bad writes to brat
2nd commit fixes bad requeueing in case of OperationalError

### CODE REVIEW

File: active/af1-dequeue-and-identify-phi.py [hunk 1]
```diff
@@ -165,30 +165,30 @@ def annotate_clinical_notes(**kwargs):
             _log_failure_and_reqeue(message, "TEMP PERSON STORAGE", run_id, blobid, hdcpupdatedate)
             continue
 
-        to_review, skip_review = split_records_by_review_status(batch_records)
-        if _review_criterion(batch_records[blobid]):
```
Comment (esilgard) [inline active/af1-dequeue-and-identify-phi.py:169]: are we getting rid of this if statement entirely? that's the placeholder for optional/percent review right?

Comment (esilgard) [reply active/af1-dequeue-and-identify-phi.py:169]: or is it duplicated logic with the review,skip_review designation?

**Review by esilgard — APPROVED** [review]
coo coo
have you tested it in the pod?

File: active/af1-dequeue-and-identify-phi.py (hunk 1 above)
Comment (WhiteAu) [reply active/af1-dequeue-and-identify-phi.py:169]: it's duplicated logic. not necessary w/ line 168 above

**Comment by WhiteAu** [comment]
> 
> 
> coo coo
> have you tested it in the pod?

Yep!

**Pull request merged**